main.py

```diff
@@ -1936,6 +1936,43 @@ async def random(message: discord.Interaction):
                 await achemb(message, "randomizer", "send")
         except Exception:
             await message.followup.send("no cats :(")
+async def light_market(message):
```

Comment:
some breathing room pls
```suggestion
            await message.followup.send("no cats :(")

async def light_market(message):
```